doc/build-instructions/Build_Instructions_V15.md

```diff
@@ -0,0 +1,627 @@
+
+A binary for the full developer kit (jdk) is built and stored in the following directory:
+
+- **build/linux-x86_64-server-release/images/jdk**
+
+    :whale: If you built your binaries in a Docker container, copy the binaries to the containers **/root/hostdir** directory so that you can access them on your local system. You'll find them in the directory you set for `<host_directory>` when you started your Docker container. See [Setting up your build environment with Docker](#setting-up-your-build-environment-with-docker).
+
+    :pencil: On other architectures the **/jdk** directory is in **build/linux-ppc64le-server-release/images** (Linux on 64-bit Power systems) and **build/linux-s390x-server-release/images** (Linux on 64-bit z Systems).
+
+    :pencil: If you want a binary for the runtime environment (jre), you must run `make legacy-jre-image`, which produces a jre build in the **build/linux-x86_64-server-release/images/jre** directory.
+
+### 5. Test
+:penguin:
+For a simple test, try running the `java -version` command.
+Change to the /jdk directory:
+```
+cd build/linux-x86_64-server-release/images/jdk
+```
+Run:
+```
+./bin/java -version
+```
+
+Here is some sample output:
+
+```
+openjdk version "15-internal" 2020-09-01
+OpenJDK Runtime Environment (build 15-internal+0-adhoc..openj9-openjdk-jdk15)
+Eclipse OpenJ9 VM (build tye-2e3d778, JRE 15 Linux amd64-64-Bit Compressed References 20200901_000000 (JIT enabled, AOT enabled)
+OpenJ9   - 2e3d778
+OMR      - a5a028d
+JCL      - 9af014f based on jdk-15+29)
+```
+
+:pencil: **OpenSSL support:** If you built an OpenJDK with OpenJ9 that includes OpenSSL support, the following acknowledgements apply in accordance with the license terms:
+
+  - *This product includes software developed by the OpenSSL Project for use in the OpenSSL Toolkit. (http://www.openssl.org/).*
+  - *This product includes cryptographic software written by Eric Young (eay@cryptsoft.com).*
+
+:penguin: *Congratulations!* :tada:
+
+----------------------------------
+
+## AIX
+:blue_book:
+
+:construction:
+This section is still under construction. Further contributions expected.
+
+The following instructions guide you through the process of building an **OpenJDK V15** binary that contains Eclipse OpenJ9 on AIX 7.2.
+
+### 1. Prepare your system
+:blue_book:
+You must install the following AIX Licensed Program Products (LPPs):
+- [xlc/C++ 16](https://www.ibm.com/developerworks/downloads/r/xlcplusaix/)
+- x11.adt.ext
+
+You must also install the boot JDK: [Java14_AIX_PPC64](https://api.adoptopenjdk.net/v3/binary/latest/14/ga/aix/ppc64/jdk/openj9/normal/adoptopenjdk).
+
+A number of RPM packages are also required. The easiest method for installing these packages is to use `yum`, because `yum` takes care of any additional dependent packages for you.
+
+Download the following file: [yum_install_aix-ppc64.txt](../../buildenv/aix/jdk15/yum_install_aix-ppc64.txt)
```

Comment:
This file needs to be copied from the jdk14 version.

Reply:
I didn't mean to refer to the jdk14 version, I meant copy the file and create a jdk15 version as part of this PR.